tests/testapp/settings.py

```diff
@@ -118,6 +118,7 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
+LOGO_URL = "s3://dc-shared-frontend-assets/images/logo_icon.svg"
```

Comment:
This will work in the test app, but it means we'd have to set it in all projects. In this instance I think it would be better to hard-code this URL in the templates. 

Also, I'm not 100% the context would actually work in the case of a 500...I think Django disables all context processors?